cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/tester/SlangTestRunner.java

```diff
@@ -252,6 +264,32 @@ private void printTestForActualRunSummary(TestCaseRunMode runMode, Map<String, S
         return resultMap;
     }
 
+    private boolean enabledByBuildMode(
+            SlangBuildMain.BuildMode buildMode,
+            SlangTestCase slangTestCase,
+            Set<String> changedExecutables,
+            Map<String, Executable> allTestedFlowModels) {
+        switch (buildMode) {
```

Comment:
I think this method can be replaced (as the `buildMode `is already validated somewhere in the begining of this flow) with `return (buildMode == BASIC) || ((buildMode == CHANGED) && isAffectedTestCase(slangTestCase, changedExecutables, allTestedFlowModels))` WDYT?